PyInstaller/building/build_main.py

```diff
@@ -219,8 +220,8 @@ def __init__(self, scripts, pathex=None, binaries=None, datas=None,
             # Create a Python module which contains the decryption key which will
             # be used at runtime by pyi_crypto.PyiBlockCipher.
             pyi_crypto_key_path = os.path.join(CONF['workpath'], 'pyimod00_crypto_key.py')
-            with open(pyi_crypto_key_path, 'w') as f:
-                f.write('key = %r\n' % cipher.key)
+            with open_file(pyi_crypto_key_path, 'w', encoding='utf-8') as f:
```

Comment:
Okay, but not necessary since the key will be `repr`-ed and thus be ascii-only.
Also when using an encoding, the first line in the file must be
```
# -*- coding: utf-8 -*-
```

Reply:
For consistence, I'd suggest sticking with UTF-8. I've added the comment.